content/sensu-go/5.20/reference/backend.md

```diff
@@ -338,6 +340,23 @@ discovery instead of the static `--initial-cluster method`
 
 ### General configuration flags
 
+| annotations|      |
+-------------|------
+description  | Non-identifying metadata to include with event data for backend assets (e.g. handler and mutator assets).
```

Comment:
I'm wondering if entity would make more sense here, because the event does not get altered with backend entity data. WDYT?
```suggestion
description  | Non-identifying metadata to include with entity data for backend assets (e.g. handler and mutator assets).
```

Reply:
Yes, I agree -- according to the notes in https://github.com/sensu/sensu-go/issues/3698 it sounds like entity is correct and not event. Thank you.